src/main/java/ch/njol/skript/effects/EffHealth.java

```diff
@@ -34,42 +18,52 @@
 import ch.njol.util.Math2;
 import org.bukkit.entity.Damageable;
 import org.bukkit.event.Event;
+import org.bukkit.event.entity.EntityDamageEvent.DamageCause;
 import org.bukkit.inventory.ItemStack;
-import org.eclipse.jdt.annotation.Nullable;
+import org.jetbrains.annotations.Nullable;
+import org.jetbrains.annotations.UnknownNullability;
 
 @Name("Damage/Heal/Repair")
-@Description("Damage/Heal/Repair an entity, or item.")
+@Description({
+	"Damage, heal, or repair an entity or item.",
+	"Servers running Spigot 1.20.4+ can optionally choose to specify a fake damage cause."
+})
 @Examples({
 	"damage player by 5 hearts",
+	"damage player by 3 hearts with fake cause fall",
 	"heal the player",
 	"repair tool of player"
 })
-@Since("1.0")
+@Since("1.0, 2.10 (damage cause)")
```

Comment:
2.10 should be `INSERT VERSION`, it'll be replaced by the proper version when it gets released